libs/sql-schema-describer/tests/test_api/mod.rs

```diff
@@ -154,6 +158,16 @@ impl SqlSchemaAssertionsExt for SqlSchema {
 
         self
     }
+
+    fn assert_namespace(&self, namespace_name: &str) -> &Self {
+        self.namespace_walker(namespace_name).or_else(|| panic!("Could not find namespace '{namespace_name}'"));
+        self
+    }
+
+    fn assert_not_namespace(&self, namespace_name: &str) -> &Self {
+        self.walk_namespaces().find(|ns| ns.name() == namespace_name).and_then::<(), _>(|_x| panic!("Found unexpected namespace '{namespace_name}'"));
```

Comment:
Is there a nicer way to write this rather than these panics?

Reply:
For tests, panics are usually more useful than `Result<_,_>` in rust because of the backtraces and line numbers. You can get the line numbers with results too of course, just not by default.

For this line, you could `.expect("Something")`, the problem is that the error message isn't lazily constructed. I usually do `.map(|_| panic!("..."))` which is pretty much the same as your solution.